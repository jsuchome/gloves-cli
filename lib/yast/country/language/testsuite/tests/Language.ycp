/**
 * tests for Language.ycp functions
 * Maintainer:	jsuchome@suse.cz
 * $Id: Language.ycp 22461 2005-03-11 15:09:25Z jsuchome $
 */
{

// testedfiles: Language.ycp

include "testsuite.ycp";

map READ = $[
    "sysconfig" : $[
	"language" : $[
	    "DEFAULT_LANGUAGE"	: "en_US",
	 ],
    ],
    "target" : $[
	"bash_output"	: $[],
        "size"		: 1,
	"yast2"		: $[],
    ],
];

TESTSUITE_INIT([READ,$[],READ], nil);
import "Pkg";
import "Language";

DUMP("GetLanguageCountry");
Language::language = "de_AT@UTF-8";
TEST(``(Language::GetLanguageCountry()), [], nil);
Language::language = "de_AT";
TEST(``(Language::GetLanguageCountry()), [], nil);
Language::language = "de";
TEST(``(Language::GetLanguageCountry()), [], nil);

Language::language = ""; // use default language

TEST(``(Language::GetLanguageCountry()), [], nil);
}
