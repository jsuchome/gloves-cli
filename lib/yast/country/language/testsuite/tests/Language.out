Read	.target.size "/etc/sysconfig/language" 1
Dump	GetLanguageCountry
Return	AT
Return	AT
Return	DE
Return	US
